schedule-2019-03-internal.md

```diff
@@ -0,0 +1,95 @@
+        -   GG added "awareness" to the bottom of the Bloom hierarchy since it's the most realistic goal for a workshop like this
+    -   Exercise: classify three learning objectives by level
+
+1.  Novice/Competent Practitioner/Expert (0:45/10:45)
+    -   <./keynote/20-novice-competent-expert.key>
+    -   Mental models
+        -   Emphasize linkages: we rarely forget completely, but we lose the ability to retrieve
+    -   Mental models don't have to be right to be useful (ball-and-spring models in chemistry)
+    -   How the graph model of human cognition expains the difference between competent and experts
+        -   Experts have more connections
+    -   Concept maps as visual representations of mental models
+    -   Using concept maps:
+        -   To design lesson
+        -   To convey information to learners
+        -   For insight into their learning
+    -   Exercise: draw and compare concept maps for ggplot2
+
+1.  Coffee (0:15/11:00)
+
+1.  Teaching as Performance I (0:30/11:30)
+    -   <./keynote/30-teaching-as-performance-1.key>
+    -   Critique the Nederbragt videos
+        -   Bad: <https://youtu.be/bXxBeNkKmJE>
+        -   Good: <https://youtu.be/SkPmwe_WjeY>
+    -   Exercise: teach in pairs (no recording)
+    -   Build rubric for use in second teaching session
+
+1.  Multimedia Learning (0:30/12:00)
+    -   <./keynote/80-multimedia-learning.key>
+    -   Exercise: create a visual to explain what the course is about
+
+## Day 2
+
+1.  The Cognitive Craft (0:45/09:45)
+    -   <./keynote/40-cognitive-craft.key>
+    -   Revisit cognitive architecture diagram and introduce 7 +/- 2
+    -   Lesson "scenes" should fit into working memory
+        -   Use exercises/discussion/elaborative examples to keep new material there long enough to be transcribed
+    -   Cognitive load theory
+        -   Intrinsic
+        -   Extraneous
+        -   Germane
+    -   Exercise: outline a faded example
+
+1.  Formative Assessment (0:45/10:30)
+    -   <./keynote/50-formative-feedback.key>
+    -   Importance of feedback loop while teaching (for instructor as well as learners)
+    -   MCQ example
+    -   Dual purposes of formative assessments (feedback loop + reinforcement)
+    -   Different kinds of formative assessment
+    -   Exercise: create MCQ and explain misconceptions it diagnoses
+
+1.  Coffee (0:15/10:45)
+
+1.  Teaching as Performance II (0:30/11:15)
+    -   <./keynote/60-teaching-as-performance-2.key>
+    -   Use the previous day's rubric
+
+1.  Motivation (0:30/11:45)
+    -   <./keynote/70-motivation.key>
+    -   Three big motivators: self-efficacy, utility, community
+    -   Three big demotivators: unpredictability, unfairness, indifference
+    -   Zone of Proximal Development
+    -   Exercise: what demotivates people in programming classes and how do we prevent/fix it?
+
+1.  Take-Home Exercise (0:15/12:00)
```

Comment:
I'd really like to recast this as an in class exercise, preferably think-pair-share. it makes a good capstone for discussing mental models and novices/experts and it also shows how teaching principles matter outside of the classroom. But I think it requires discussion and feedback from the instructors.

Reply:
What do you think of the revised order?

Reply:
+ I like moving formative assessment up to Two ways to teach and I'm happy to have you teach that one
+ I like finishing with a reflective R Markdown exercise, like we discussed yesterday
+ I like how the exercises focus more on "unofficial" teaching situations

- I strongly feel that cognitive craft should be moved to immediately after novice/competent practitioner/expert.
- I think you are a better choice to teach novice/competent practitioner/expert
- I'd prefer to move the davio discussion up to Day 1 to be an application of the novice/competent practitioner/expert model